pkg/chunk/cassandra/storage_client.go

```diff
@@ -89,7 +89,7 @@ func (cfg *Config) Validate() error {
 	return nil
 }
 
-func (cfg *Config) session(name string) (*gocql.Session, error) {
+func (cfg *Config) session(name string, registerer prometheus.Registerer) (*gocql.Session, error) {
```

Comment:
```suggestion
func (cfg *Config) session(name string, reg prometheus.Registerer) (*gocql.Session, error) {
```

I noticed you used `reg` for the dynamodb package and `registerer` here. I don't have a preference but we should try to be consistent if we can.